code/game/gamemodes/game_mode.dm

```diff
@@ -516,3 +516,217 @@
+	var/text
+	text += "<font color='red' size=3><center><b>You are the [antagonist].</b></center></font><br><br>"
+	switch(antagonist)
+		if("traitor")
+			text += "<center>You are an agent of the Syndicate aboard [station_name()].</center>"
+			text += "<center>You are to accomplish the following objectives assigned to you by your employers:</center><br>"
+			var/objCount = 1
+			for(var/datum/objective/objective in M.objectives)
+				text += "<center><b>Objective #[objCount]:</b> [objective.explanation_text]</center>"
+				objCount++
+			text += "<br><center>You have also been supplied with an uplink cunningly hidden in your PDA or headset. This uplink contains 20 telecrystals and allows you to purchase exclusive equipment \
+			to aid in completing your objectives. In addition, you have been given a set of code words to identify other agents. Use these in conversation but be aware that anyone may be a potential foe. \
+			</center><br><br>"
+			text += "<center><b>Code Phrases:</b> <font color='red'>[syndicate_code_phrase]</font></center>"
+			text += "<center><b>Code Responses:</b> <font color='red'>[syndicate_code_response]</font></center><br><br>"
+			text += "<center>Accomplish your objectives at all costs!</center>"
+			text += "<center><i><b>Wiki Page:</b></i> <a href='[config.wikiurl]/Traitor'>Click Here</a></center>"
+
+
+		if("double agent")
+			text += "<center>You are a Syndicate double agent aboard [station_name()].</center>"
+			text += "<center>For some reason or another, you have been assigned to kill another Syndicate agent. However, be wary, as a third double agent is also trying to kill <i>you!</i> You will need \
+			to kill your quarry while avoiding death yourself. Your objectives:</center><br>"
+			var/objCount = 1
+			for(var/datum/objective/objective in M.objectives)
+				text += "<center><b>Objective #[objCount]:</b> [objective.explanation_text]</center>"
+				objCount++
+			text += "<br><center>You have also been supplied with an uplink cunningly hidden in your PDA or headset. This uplink contains 20 telecrystals and allows you to purchase exclusive equipment \
+			to aid in completing your objectives. In addition, you have been given a set of code words to identify other agents. Use these in conversation but be aware that anyone may be a potential foe. \
+			</center><br><br>"
+			text += "<center><b>Code Phrases:</b> <font color='red'>[syndicate_code_phrase]</font></center>"
+			text += "<center><b>Code Responses:</b> <font color='red'>[syndicate_code_response]</font></center><br><br>"
+			text += "<center>Eliminate your quarry and avoid death at all costs!</center>"
+			text += "<center><i><b>Wiki Page:</b></i> <a href='[config.wikiurl]/Traitor'>Click Here</a></center>"
+
+
+		if("changeling")
+			text += "<center>We are a changeling, an alien lifeform aboard [station_name()].</center>"
+			text += "<center>We are employed by the Syndicate and must accomplish the following objectives they wish of us:</center><br>"

```

Comment:
I don't like the pop-up. Roundstart is usually a rush to get the good shit
while stopping other people from getting that good shit. I know you can
just minimise it, but you're still going to have to pull it up to read it
and have it obscure your vision at some point. Is it really necessary? Can
you just add the wiki link to the message you get in the text field instead?

On 30 June 2015 at 18:41, Remie Richards <notifications@github.com> wrote:

> In code/game/gamemodes/game_mode.dm
> <https://github.com/tgstation/-tg-station/pull/10281#discussion_r33544087>
> :
>
> > +			var/objCount = 1
> > +			for(var/datum/objective/objective in M.objectives)
> > +				text += "<center><b>Objective #[objCount]:</b> [objective.explanation_text]</center>"
> > +				objCount++
> > +			text += "<br><center>You have also been supplied with an uplink cunningly hidden in your PDA or headset. This uplink contains 20 telecrystals and allows you to purchase exclusive equipment \
> > +			to aid in completing your objectives. In addition, you have been given a set of code words to identify other agents. Use these in conversation but be aware that anyone may be a potential foe. \
> > +			</center><br><br>"
> > +			text += "<center><b>Code Phrases:</b> <font color='red'>[syndicate_code_phrase]</font></center>"
> > +			text += "<center><b>Code Responses:</b> <font color='red'>[syndicate_code_response]</font></center><br><br>"
> > +			text += "<center>Eliminate your quarry and avoid death at all costs!</center>"
> > +			text += "<center><i><b>Wiki Page:</b></i> <a href='[config.wikiurl]/Traitor'>Click Here</a></center>"
> > +
> > +
> > +		if("changeling")
> > +			text += "<center>We are a changeling, an alien lifeform aboard [station_name()].</center>"
> > +			text += "<center>We are employed by the Syndicate and must accomplish the following objectives they wish of us:</center><br>"
>
> "Supervisors: The Syndicate, probably"
> wiki gives no hints.
>
> To my knowledge they are part of the Syndicate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10281/files#r33544087>.
>